lib/vdsm/storage/mpathconf.py

```diff
@@ -169,6 +377,19 @@ def read_blacklist():
     return wwids
 
 
+def configure_multipathd():
+    """
+    Set up multipath daemon configuration to the known and supported state.
+    The original configuration, if any, is saved.
+    """
+    backup = fileUtils.backup_file(CONF_FILE)
+    if backup:
+        log.info("Previous multipath.conf copied to %s\n" % backup)
```

Comment:
Again this log will be hidden in vdsm-tool it should go to stdout, but we don't want
to do this here.

We can return the backup path and let the caller do the logging in the configurator.